src/components/RouteWrapper.tsx

```diff
@@ -428,6 +429,13 @@ export const ROUTES: RouteProperties[] = [
         description: "A tool used for capturing screenshots and gathering information from web servers",
         category: "Web Application Testing",
     },
+    {
+        name: "Exif",
+        path: "/tools/Exif",
+        element: <Exif />,
+        description: "A tool used to extract metadata from different file types",
+        category: "Information Gathering and Analysis",
+    },
```

Comment:
Routewrapper tools should be in alphabetical order. Please place Exif above EyeWitness.